modules/ztp-sno-du-tuning-the-performance-patch.adoc

```diff
@@ -8,4 +8,24 @@
 
 {sno-caps} clusters that run DU workloads require additional performance tuning configurations necessary for high-performance workloads. The following example `Tuned` CR extends the `Tuned` profile:
 
-include::snippets/ztp-performance-patch.adoc[]
+.Recommended extended Tuned profile configuration
+[source,yaml]
+----
+include::snippets/ztp_TunedPerformancePatch.yaml[]
+----
+
+.Tuned CR options for {sno} clusters
+[cols=2*, width="90%", options="header"]
+|====
+|Tuned CR field
+|Description
+
+|`spec.profile.data`
+a|* The `include` line that you set in `spec.profile.data` must match the associated `PerformanceProfile` CR name.
+For example, `include=openshift-node-performance-${PerformanceProfile.metadata.name}`.
+
+* The `cmdline_crash` CPU set must match the `isolated` set in the associated `PerformanceProfile` CR.
```

Comment:
This field has been removed from the reference CR

Reply:
removed this line